.eslintrc

```diff
@@ -0,0 +1,13 @@
+{
+  "parserOptions": {
+      "ecmaVersion": 2017
+  },
+
+  "env": {
+      "es6": true
+  },
+
+  "rules": {
+    "semi": ["error", "always"]
```

Comment:
Let's configure the linter so that it uses prettier's formatting 🙂 Take a look at https://github.com/prettier/eslint-plugin-prettier